SCE-Development/Core-v4
# 2D and 3D printing pages

updated the designs of the 2D and 3D printing pages

3d printing:
![image](https://user-images.githubusercontent.com/36345325/156988588-6e13298e-2fc9-43d3-a3e2-8eebcc7ea3b7.png)

2d printing:
![image](https://user-images.githubusercontent.com/36345325/156988640-4c8d0e80-5173-4ffa-bced-9b7b0febf1c0.png)

### How to test
- run frontend, backend and mongodb
- sign in as a user with at least member priviledge
- navigate to the 2d and 3d printing pages with expected visuals above
